packages/SwingSet/src/kernel/deviceManager.js

```diff
@@ -38,7 +38,11 @@ export default function makeDeviceManager(
    *    or is otherwise invalid.
    */
   function mapDeviceSlotToKernelSlot(devSlot) {
-    assert.equal(devSlot, `${devSlot}`, details`non-string devSlot: ${devSlot}`);
+    assert.equal(
```

Comment:
this btw was an experiment:  ```assert.equal(actual, expected, details`...`)``` vs. ```assert(actual === expected, details`...`)```. I realized a little ways into rewriting asserts that we want to keep `typeof` comparison as a conditional because then linting rules will ensure we are only testing for valid typeof strings. We could add a `assert.type` assertion, but a) I don't think there's an equivaletn in nodeJS assert (a weak but still notable negative), and b) it loses the lint check.

Are there other expressions that would be best left as conditionals?  The `assert.equal` form is good if you aren't going to actaully provide a details message, because it provides a better default details.